.changeset/six-penguins-eat.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
The file name : six-penguins-eat seems odd

Reply:
The file names are generated randomly by "changesets". The files are deleted when the changeset is consumed by a new release.